README.md

```diff
@@ -9,6 +9,15 @@ For running Ansible.
 * `ONBUILD` versions of each distro
 * Ansible `2.3` versions of each, eg: `2.3-alpine`
 
+### Quick Start
+1. Download and install Docker community edition - https://www.docker.com/community-edition
+2. Clone the ansible role or playbook repo for testing.
```

Comment:
Could we add a point 3 and maybe 4 here?

> Dependencies are automatically sourced on-the-fly from requirements.yml for Ansible Galaxy, requirements.txt for PyPI, and system_packages.txt for yum or apk. 

> Supports Ansible 2.3 and 2.4 on Alpine and CentOS 7.

Reply:
Sorry, I don't understand. I added the Quick Start section so folks could see the few steps they need to execute before using the image. 

These other "feature" aspected, I added in the initial summary. Does that work?

Reply:
That line about dependencies is required to start quickly though. If someone pulled and ran a role that wasn't created with my cookiecutter template, they wouldn't have a test.yml to run. The "Supports" line should go *somewhere*, but I wasn't sure where. I'll leave that up to you.

Reply:
Oh also, please add proper caps on Ansible